test/e2e/traffic/trafficrec.go

```diff
@@ -59,6 +60,16 @@ func (tr *SimpleTrafficRec) RecordStats(stats TrafficStats) {
 	tr.stats.ServerReceived += stats.ServerReceived
 }
 
+func (tr *SimpleTrafficRec) RecordClientAddr (addr string) {
+	tr.Lock()
+	defer tr.Unlock()
+    tr.clientAddr = addr
+}
+
+func (tr *SimpleTrafficRec) ShowClientAddr () string {
```

Comment:
just `ClientAddr`; also, locking should be done in this func

Reply:
Fixed, thanks